From fa831c769d58bfedd9584019c8ce85879b4c017e Mon Sep 17 00:00:00 2001
From: Oliver Beddows <oliver.beddows@googlemail.com>
Date: Sat, 21 Nov 2015 16:39:14 +0000
Subject: [PATCH 3/4] Patching preparser.h.

---
 deps/v8/src/preparser.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/v8/src/preparser.h b/deps/v8/src/preparser.h
index 8a93258..827ed7c 100644
--- a/deps/v8/src/preparser.h
+++ b/deps/v8/src/preparser.h
@@ -437,7 +437,7 @@ class ParserBase : public Traits {
   void ReportMessageAt(Scanner::Location location, const char* message,
                        bool is_reference_error = false) {
     Traits::ReportMessageAt(location, message,
-                            reinterpret_cast<const char*>(NULL),
+                            reinterpret_cast<const char*>(0),
                             is_reference_error);
   }
 
-- 
2.6.3

