From ff6fc654fd11d605fe5165c8fa06bd10c022e764 Mon Sep 17 00:00:00 2001
From: Oliver Beddows <oliver.beddows@googlemail.com>
Date: Sat, 21 Nov 2015 16:39:31 +0000
Subject: [PATCH 4/4] Patching unique.h.

---
 deps/v8/src/unique.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/deps/v8/src/unique.h b/deps/v8/src/unique.h
index ffc659f..0b6c343 100644
--- a/deps/v8/src/unique.h
+++ b/deps/v8/src/unique.h
@@ -108,7 +108,7 @@ class Unique {
 
   // TODO(titzer): this is a hack to migrate to Unique<T> incrementally.
   static Unique<T> CreateUninitialized(Handle<T> handle) {
-    return Unique<T>(reinterpret_cast<Address>(NULL), handle);
+    return Unique<T>(NULL, handle);
   }
 
   static Unique<T> CreateImmovable(Handle<T> handle) {
@@ -167,7 +167,7 @@ class PrintableUnique : public Unique<T> {
 
   // TODO(titzer): this is a hack to migrate to Unique<T> incrementally.
   static PrintableUnique<T> CreateUninitialized(Zone* zone, Handle<T> handle) {
-    return PrintableUnique<T>(zone, reinterpret_cast<Address>(NULL), handle);
+    return PrintableUnique<T>(zone, NULL, handle);
   }
 
   static PrintableUnique<T> CreateImmovable(Zone* zone, Handle<T> handle) {
-- 
2.6.3

