From 74da03e5df83770d554604391ce2858f815da7d2 Mon Sep 17 00:00:00 2001
From: Oliver Beddows <oliver@lisk.io>
Date: Mon, 10 Oct 2016 09:13:55 +0000
Subject: [PATCH] Patching common.gypi.

---
 common.gypi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/common.gypi b/common.gypi
index 1984fa8..8a57cea 100644
--- a/common.gypi
+++ b/common.gypi
@@ -182,7 +182,6 @@
       [ 'OS in "linux freebsd openbsd solaris android"', {
         'cflags': [ '-Wall', '-Wextra', '-Wno-unused-parameter', ],
         'cflags_cc': [
-          '-fno-delete-null-pointer-checks',
           '-fno-exceptions',
           '-fno-rtti',
         ],
-- 
2.8.1

