From 0d62b4af7289e681f23c584c764bce3333ca1bd0 Mon Sep 17 00:00:00 2001
From: Oliver Beddows <oliver.beddows@googlemail.com>
Date: Fri, 27 Nov 2015 16:31:30 -0500
Subject: [PATCH] Hard coding arm_version variable.

---
 configure | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/configure b/configure
index 66cb312..b4f919c 100755
--- a/configure
+++ b/configure
@@ -490,13 +490,7 @@ def configure_arm(o):
   else:
     arm_float_abi = 'default'
 
-  if is_arch_armv7():
-    o['variables']['arm_version'] = '7'
-  elif is_arch_armv6():
-    o['variables']['arm_version'] = '6'
-  else:
-    o['variables']['arm_version'] = 'default'
-
+  o['variables']['arm_version'] = '7'
   o['variables']['arm_fpu'] = 'vfpv3'  # V8 3.18 no longer supports VFP2.
   o['variables']['arm_neon'] = int(is_arm_neon())
   o['variables']['arm_thumb'] = 0      # -marm
-- 
2.1.4

