From 8ee0d9edbbe9dfaf35061ee759eec7e44ad030fe Mon Sep 17 00:00:00 2001
From: Oliver Beddows <oliver.beddows@googlemail.com>
Date: Thu, 26 Nov 2015 19:23:06 -0500
Subject: [PATCH 1/2] Fixing v8 armv6 processor detection.

---
 deps/v8/src/base/cpu.cc | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/deps/v8/src/base/cpu.cc b/deps/v8/src/base/cpu.cc
index adce69d..ec946dc 100644
--- a/deps/v8/src/base/cpu.cc
+++ b/deps/v8/src/base/cpu.cc
@@ -377,6 +377,15 @@ CPU::CPU() : stepping_(0),
       }
       delete[] processor;
     }
+
+    // elf_platform moved to the model name field in Linux v3.8.
+    if (architecture_ == 7) {
+      char* processor = cpu_info.ExtractField("model name");
+      if (HasListItem(processor, "(v6l)")) {
+        architecture_ = 6;
+      }
+      delete[] processor;
+    }
   }
 
   // Try to extract the list of CPU features from ELF hwcaps.
-- 
2.1.4

