From 0eb74babb64bcb6ba6c6e3c2f8307bc1ec019636 Mon Sep 17 00:00:00 2001
From: Oliver Beddows <oliver.beddows@googlemail.com>
Date: Thu, 26 Nov 2015 19:23:59 -0500
Subject: [PATCH 2/2] Hard coding arm_version and arm_fpu variables.

---
 configure | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/configure b/configure
index 66cb312..50e6506 100755
--- a/configure
+++ b/configure
@@ -490,14 +490,8 @@ def configure_arm(o):
   else:
     arm_float_abi = 'default'
 
-  if is_arch_armv7():
-    o['variables']['arm_version'] = '7'
-  elif is_arch_armv6():
-    o['variables']['arm_version'] = '6'
-  else:
-    o['variables']['arm_version'] = 'default'
-
-  o['variables']['arm_fpu'] = 'vfpv3'  # V8 3.18 no longer supports VFP2.
+  o['variables']['arm_version'] = '6'
+  o['variables']['arm_fpu'] = 'vfpv2'
   o['variables']['arm_neon'] = int(is_arm_neon())
   o['variables']['arm_thumb'] = 0      # -marm
   o['variables']['arm_float_abi'] = arm_float_abi
-- 
2.1.4

